### ISSUE TITLE
[Effect Suggestion] Danger Close

### ISSUE THREAD

**Issue by SeanMirrsen** [gta-chaos-mod/ChaosModV#2025]
Duration: Instant, or very short (custom, a second or two at most)

High up in the sky, a large cluster of RPG projectiles is fired down towards the player, similarly to the Airstrike Incoming effect.
However, this barrage of projectiles, fired like a shotgun blast or a very short rapid-fire burst, follows two very specific rules.
1. The projectiles are grouped over a much smaller area, resulting in a much more dense area of destruction.
2. None of the projectiles are aimed within the roughly ~10 meter radius around the player, with some deviation.

The practical upshot is a very condensed 'airstrike' that hits everything around the player, *excluding the player* - on the condition that within the scant few seconds between the effect activation and the projectiles hitting the ground, the player does not move - or manages to quickly double back into the area (which, for convenience, could be highlighted).

Trivia: "Danger Close" is military speak for an airstrike or artillery strike being ordered directly onto, or very close - dangerously close - to your own position, with the intent on hitting the enemy that is close to taking it over.

**Comment by TerrizX** [comment]
Ain't tryna knock the idea, but this just seems too similar to what Airstrike Inbound already does. In game it would look very similar.

**Comment by SeanMirrsen** [comment]
> 
> 
> Ain't tryna knock the idea, but this just seems too similar to what Airstrike Inbound already does. In game it would look very similar.

Not quite. Airstrike Inbound is an ambient threat, of sorts. It's not something you can avoid if it happens to spawn a rocket in the right place, so it's something you tend to ignore and just press on hoping it won't kill you.

This, is a one-time *lethal threat*, but one that is predictable and avoidable. The barrage of fire *will* kill you if you're in the wrong place, but you will be safe (even if your mission companions won't necessarily be) as long as you're either in the 'safe zone' or driving/flying fast enough to be out of the threat area by the time the projectiles land.
